src/panels/lovelace/cards/hui-picture-card.ts

```diff
@@ -79,18 +80,20 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
       return html``;
     }
 
+    const clickable = Boolean(
+      this._config.tap_action || this._config.hold_action
```

Comment:
```suggestion
      this._config.tap_action || this._config.hold_action || this._config.double_tap_action
```

Reply:
probably a `hasActions(this._config)` function would be best for re-usability by other elements

Reply:
Right. Also I didn't see you did this in #4494 so I'll remove this again.